### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ethangj**
@mlocher to review

### CODE REVIEW

File: codeship-steps.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   steps:
     - name: Test Deployment
       service: deployment
-      command: /deploy/test/deploy_to_google.sh
+      command: /test/deploy_to_google.sh
```
Comment (mlocher) [inline codeship-steps.yml]: The `deployment` folder inside the repository is mounted as a volume to `/deploy` (inside the container) via https://github.com/codeship-library/google-cloud-utilities/blob/master/codeship-services.yml#L9

File: deployment/test/deploy_to_google.sh [hunk 2]
```diff
@@ -6,7 +6,12 @@ cluster_name="codeship-gcloud-test-${date}"
 
 codeship_google authenticate
 
-echo "Starting a small cluster with a single instance"
+# echo "Setting context if cluster exists"
```
Comment (mlocher) [inline deployment/test/deploy_to_google.sh]: Does it have any negative effects if we set the context in any case?

Comment (ethangj) [reply deployment/test/deploy_to_google.sh]: @mlocher It's commented out, so we're not doing it by default

**Comment by mlocher** [comment]
@ethangj could you take a look at this PR again. Do you think we should merge that, or can we close the PR?

**Pull request closed without merging**